doc/source/data/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -216,22 +224,23 @@
    },
    "outputs": [],
    "source": [
-    "from torchvision.models import ResNet18_Weights\n",
+    "from torchvision.models import ResNet152_Weights\n",
     "from torchvision import transforms\n",
     "from torchvision import models\n",
     "\n",
-    "weights = ResNet18_Weights.IMAGENET1K_V1\n",
+    "weights = ResNet152_Weights.IMAGENET1K_V1\n",
```

Comment:
Can we update the title of the example in 

- doc/source/ray-overview/examples.rst, L216
- doc/source/data/examples.rst, L33